openapi-spec/swagger.yaml

```diff
@@ -141,6 +141,25 @@ paths:
           description: Forbidden
         '500':
           $ref: '#/responses/HTTPStatus500'
+  '/v1beta/{projectId}/availabilityZones':
+    parameters:
+      - $ref: '#/parameters/projectId'
+    get:
+      tags:
+        - Pool
+      description: Lists availabilityZones without repeating for all block storage pool.
```

Comment:
Lists availabilityZones without repeating for all block storage pool. -> Lists availability zones.

Reply:
In addition, change the tag Pool to Availability Zone